package.json

```diff
@@ -42,13 +43,15 @@
     "yalc:all": "for d in packages/*/; do echo $d; cd $d; yalc push --replace --sig; cd '../../'; done"
   },
   "devDependencies": {
+    "@actions/core": "^1.10.1",
```

Comment:
Intentionally kept these with the idea being that CI cache has a higher chance of being used.